networks.json

```diff
@@ -4,6 +4,78 @@
       "links": {},
       "address": "0x0905Ab807F8FD040255F0cF8fa14756c1D824931",
       "transactionHash": "0x374c62b4eaf0c5d1c8b94a7d0c4564ecb077fe9b3ac50b08decea3b2e972ca43"
+    },
+    "4": {
+      "events": {},
+      "links": {},
+      "address": "0xc8e7354a5455b99a0c42c89ee79b4cca1804e957",
+      "transactionHash": "0x6d57ee69ad1b36e9d4e4e5032efade4002dc87e3d60dfeb0bfe97abe452df17f"
+    },
+    "42": {
+      "events": {},
+      "links": {},
+      "address": "0x8bdd53a53c53fbc40d38231d6219f8cb7b59a704",
+      "transactionHash": "0x6c4ae0abb0355a44b88d90e13a3359cc7d28215afdd65928a898bca1d228d337"
+    }
```

Comment:
These are the Migration.sol addresses not `BridgedTokenOWL`

Reply:
I coppied this directly from the lost info in #33...

Reply:
I see, there is another line at the top I missed.

Reply:
Fixed.